### PR TITLE
[14.0] [mig] pos_hide_banknote_button

### PR DESCRIPTION
**Pull request by dsolanki-initos**

### CODE REVIEW

**Comment by dsolanki-initos** [comment]
@chienandalu can you please review this PR?

File: pos_hide_banknote_button/static/src/css/pos_hide_banknote_button.css
Comment (legalsylvain) [inline pos_hide_banknote_button/static/src/css/pos_hide_banknote_button.css:18]: I think that this new way is less predictive that the previous one. If other OCA modules are adding / altering numpad buttons, it could generate conflicts. 

don't you think ? 

Not sure how fix it thought.

Otherwise, lgtm, tested on runbot.

Comment (dsolanki-initos) [reply pos_hide_banknote_button/static/src/css/pos_hide_banknote_button.css:18]: Yes but there is no other class which I can use to apply the css. If you have the otherway let me know about it.

**Review by fshah-initos — APPROVED** [review]
LGTM. Functional Test and Code review

**Comment by brendapaniagua** [comment]
LGTM!! Functional review

**Comment by dsolanki-initos** [comment]
It would be greate if we could merge this PR.

**Comment by dsolanki-initos** [comment]
Hi @chienandalu can you please merge this PR?

**Comment by dsolanki-initos** [comment]
Hi @chienandalu it would be great if you could merge this PR.

**Comment by legalsylvain** [comment]
Hi @dsolanki-initos, fshah-initos

First, thanks for porting many OCA / pos (and other repos) modules to the 14.0.
Some rules regarding reviews and merge : 

- To be accepted, a PR should have two approvals by independent reviewers (preferably from another company) during 5 days. See OCA convention : https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/oca_module_lifecycle_development_status.rst
- Repo maintainers are receiving notification from github where new PR are created, or updated. No needs to ping them each week ! 
- For the OCA to work, people need to read twice as many PRs as they offer, preferably from people from other companies, to discover other manner to work and develop. So, while waiting for your PRs to be validated, you can take the opportunity to review others.

I'll take a look in a few weeks on your PRs, but for the time being, i have no time for the review. I'm currently working on 12.0 revision. Recent version (V14) introduced a lot of changes in the framework and the structure (introduction of owl, etc...) I need to understand these changes a little more to make valid code reviews.

Regards.

**Comment by dsolanki-initos** [comment]
Hi @legalsylvain,

I hope you are doing good!

I've counted two reviews for this PR and then requested the maintainer for Merge. For this PR we got two reviews one from **Foram-Initos** and second **brendapaniagua** I don't know we counted the same or not.


We understand that repo maintainers are busy with other works. Repo maintainers are receving notfications but we are not getting any response so we tag them as we need to merge our PR soon.


We do reviews for other's PR. We have not measured for have many PR's we have done review. If you want we can do so.
Do let us know the PR's where the reviews are needed so that we can add reviews over there.

Thank you!

Regards,
Dhara-Initos

**Comment by dsolanki-initos** [comment]
It would be nice if we get the reviews here.

**Comment by legalsylvain** [comment]
@dsolanki-initos. 

> I've counted two reviews for this PR and then requested the maintainer for review. For this PR we got two reviews one from Foram-Initos and second brendapaniagua I don't know we counted the same or not.

@brendapaniagua : please use the approve button of github to make your approval visible, and catchable by oca bot.

Well, @dsolanki-initos, it was a general remarks, regarding all the new PR that initos is doing on OCA/pos. For exemple : 

- #671 by @dsolanki-initos 
image

- #651 by @dsolanki-initos
image

- #653 by @fshah-initos
image


People who are interested in pos (maintainers or not) subscribe to the repo, and thus receive a notification (or mail) when a new PR is created. (but they also receive one for each new post!). So, thanks for reducing the noise on this repo !

> we are not getting any response so we tag them as we need to merge our PR soon.

In the meantime, having unmerged PR is a normal situation, and you can continue to work with the branch, without beeing blocked.


/ocabot merge patch

**Pull request merged**
